apps/condo/domains/user/schema/ConfirmPhoneAction.js

```diff
@@ -63,6 +63,23 @@ const checkLock = async (phone, action) => {
     }
 }
 
+const getMobilePlatform = (userAgent) => {
+    const isMobile = !!userAgent.match(/mobile/i)
+    const isIpad = !!userAgent.match(/ipad/i)
+    const isIPhone = !!userAgent.match(/iphone/i)
+    const isIPod = !!userAgent.match(/ipod/i)
+    const isAndroid = !!userAgent.match(/android/i)
+    if (isMobile) {
+        if (isIpad || isIPhone || isIPod) {
+            return 'iOS'
+        }
+        if (isAndroid) {
+            return 'Android'
+        }
+    }
+    return ''
+}
+
```

Comment:
Why not to use third-party libraries to parsing userAgent string, like [ua-parser-js](https://www.npmjs.com/package/ua-parser-js) or [bowser](https://www.npmjs.com/package/bowser)?